backend/lib/components/CDN/index.js

```diff
@@ -12,6 +12,14 @@ async function uploadUserAvatar(id, file) {
   return _uploadAvatar("users", id, file);
 }
 
+async function deleteOrgAvatars(id) {
```

Comment:
We don't want to call S3 to delete the images directly because if a user has a lot of images, it will be slow and will block the thread, so it will result in a bad user experience. Instead, to make it asynchronous, we should put a message onto an SQS queue with the user ID or org ID for which we want to delete avatar images, and then the SQS messages would be processed by a Lambda to delete the images. So this code would have to move to a Lambda.

I discussed this with @CamilaArias in a Slack DM the other day but I was under the impression that it was sufficient to just delete the `photo` attribute from the user and org for now, and implement the delete avatar Lambda later.

Reply:
An alternative solution is rather than keep all the avatar images that were uploaded by a particular user, when the user updates their avatar photo, we should just delete the currently used avatar photo in S3 and then upload the new avatar photo. This way we would only have one avatar image stored per user in S3. I don't see a good reason for keeping all their old avatar images in S3 - the reason why we did that in the first place was we needed to bust the CDN cache by appending a timestamp to the image file name (the image file name is simply `${userId}-${unixTimestamp}`). Otherwise, the avatar image would not appear to change on the frontend because the CDN is still serving the old image file. 

This should be easy to implement by taking the image file name currently stored in the `photo` field, and then delete that file from S3.

Reply:
After discussing with @ddveloper in #1981, we decided that we should revert this code for deleting from S3 since this is outside the scope of #1967, and we will implement it later.

Reply:
Thanks @mannykary 👍 super clear